### DISCUSSION TITLE
Rescaling the colourkey

### DISCUSSION THREAD

**Discussion by bltsai-UCLA** [uclahs-cds/public-R-BoutrosLab-plotting-general#61] [Q&A] [answered]
Is there a way to set the limits on the colourkey for heatmaps and/or dotmaps? For example, the colourkey values range from 0-100 here, which is not useful. I'd like to be able to set the upper limit to something lower like 10.
![image](https://user-images.githubusercontent.com/66393152/202337184-59888877-7fe1-4655-b382-4d9a9d02c24a.png)

**Comment by pboutros** [comment, marked as answer]
I think you just want the at parameter?

________________________________

UCLA HEALTH SCIENCES IMPORTANT WARNING: This email (and any attachments) is only intended for the use of the person or entity to which it is addressed, and may contain information that is privileged and confidential. You, the recipient, are obligated to maintain it in a safe, secure and confidential manner. Unauthorized redisclosure or failure to maintain confidentiality may subject you to federal and state penalties. If you are not the intended recipient, please immediately notify us by return email, and delete this message from your computer.
